target/linux/octeon/base-files/lib/upgrade/platform.sh

```diff
@@ -38,20 +44,34 @@ platform_do_flash() {
 	local rootfs=$4
 
 	mkdir -p /boot
-	mount -t vfat /dev/$kernel /boot
 
-	[ -f /boot/vmlinux.64 -a ! -L /boot/vmlinux.64 ] && {
+	if [ $board = "itus,shield-router" ]; then
+	   # mmcblk1p1 (fat) contains all ELF-bin images for the Shield
```

Comment:
You are mixing tab and space indent here. Please use tabs only as done in that file so far.

Reply:
Will correct